README.md

```diff
@@ -22,26 +23,29 @@ chmod -R o+w input_files
 ## Execution
 ```
 docker-compose -f docker-compose-CeleryExecutor.yml up -d
-Open Airflow UI: http://<server_ip>:8080/admin/
-Toggle switch to enable/start the DAG
+
+Open Airflow UI: http://<docker_host_ip>:8080/admin/
+
+Toggle switch from "Off" to "On" to enable the SandboxSubmision DAG
+
 ```
 
 ## Test Run:
 ```
 Open commmand prompt or terminal and run:
 docker container ps
 
-Note the CONTAINER_ID of the container running airflow webserver
+Note the CONTAINER_ID of the container running airflow webserver. The airfow webserver will have "/entrypoint.sh webserver" in the COMMAND column of the docker container ps output.
 
 Execute this command below to trigger the DAG with the provided sample study:
 docker exec -it <REPLACE_WITH_CONTAINER_ID_FROM_ABOVE> python /usr/local/airflow/.local/bin/run_qc.py --host_port localhost:8080 -r input_files -o output_files -m input_files/user_study/metadata.json -s user_study
 
-Open Airflow UI: http://<server_ip>:8080/admin/ to check status of DAG run
+Open Airflow UI: http://<docker_host_ip>:8080/admin/airflow/graph?dag_id=SandboxSubmission/ to check status of DAG run
 ```
 
 ## Stop Docker Containers
 ```sh
-docker-compose -f docker-compose-CeleryExecutor.yml stop
+docker-compose -f docker-compose-CeleryExecutor.yml down
```

Comment:
What's the difference between the two?

Reply:
I see that `down` deletes the stopped containers vs `stop` just stops the running containers. Do we have to delete the stopped containers?

Reply:
My recommendation is to delete the containers. It keeps the host machine cleaner. It also allow the gaptools image to be deleted without finding the remnant of a stopped container. Since the implemenation is "stateless" there is no need to save the state of the containers.